app/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```

Comment:
I am assuming we might want to use [ProGuard](https://developer.android.com/studio/build/shrink-code.html#shrink-resources) but I haven't really looked into it yet. It allows you to make your list of resources shorter somehow.